components/playground/main.go

```diff
@@ -122,6 +122,7 @@ Examples:
   $ tiup playground nightly --without-monitor       # Start a local cluster and disable monitor system
   $ tiup playground --pd.config ~/config/pd.toml    # Start a local cluster with specified configuration file
   $ tiup playground --db.binpath /xx/tidb-server    # Start a local cluster with component binary path
+  $ tiup playground -T xx                           # Start a local cluster with data dir named 'xx' and uncleaned after exit
```

Comment:
```suggestion
  $ tiup playground --tag xx                           # Start a local cluster with data dir named 'xx' and uncleaned after exit
```

It's better to unify the style in the example and the tip message below.